tests/enableCache/perspectives.js

```diff
@@ -126,7 +126,6 @@ describe(`tests/enableCache/perspectives.js, api: GET ${path} >`, () => {
       }
 
       expect(res.body).to.have.length(ONE);
-      expect(res.body[ZERO].lensId).to.not.exist;
```

Comment:
This is going to be added back after the TODO is done, right? Would it be better to comment this out until then?

Reply:
yeap